### ISSUE TITLE
keyboard navigation does not work with rtl language

### ISSUE THREAD

**Issue by efef10** [Hacker0x01/react-datepicker#1663]
I have a problem with the keyboard arrows with rtl locale language, when I press the left key, it acts as right, and vice versa...

is there any solution?
Thanks!
